addon/components/frost-list-core.js

```diff
@@ -39,7 +36,6 @@ const FrostList = Component.extend(PropTypeMixin, {
   },
 
   alwaysUseDefaultHeight: false,
-  defaultHeight: 45,
```

Comment:
I put it here original to fix a s&m regression, and now it looks ok without it after I update the s&m from 0.4.x to 0.5.x, instead it produces a wired css issue.  There's still a chain in template that user can set `defaultHeight`via component interface.